modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -40,7 +40,7 @@ If the storage type is `NFS`, and you want to scale up the registry Pod by setti
 ----
 # cat /etc/exports
 /mnt/data *(rw,sync,no_wdelay,no_root_squash,insecure,fsid=0)
-sh-4.3# exportfs -rv
+sh-4.4# exportfs -rv
```

Comment:
And this file :)

Reply:
See https://github.com/openshift/openshift-docs/pull/21456/files#r414132734